### ISSUE TITLE
Plugin Unresponsive in Home App - Works in HomeBridge

### ISSUE THREAD

**Issue by weissk93** [node-alarm-dot-com/homebridge-node-alarm-dot-com#132]
Alarm.com accessories become unresponsive shortly after adding bridge to Home app. Accessories work in HomeBridge interface. Is anyone else having this issue? 

I tried restarting modem, router, and primary home hub. Also, I have removed and added the plugin/bridge multiple times. Last, I tried with both my Hoobs box and then my Mac Mini using HomeBridge. Note these are all on the latest software versions.

This issue is specific to the Alarm.com plugin and not the other plugins that I have.

From my standpoint, there is nothing in the logs that indicate any issues.

This issue started for me a few days ago and I went down the rabbit hole trying to diagnose it. Tried my best to get the required logs, I am a bit of an amateur when it comes to this. If you need anything else, please walk me through it.

* **Node.js Version**:  v20.9.0
* **Homebridge Version**: v1.8.3
* **Operating System**: MacOS

Thanks

**Comment by weissk93** [comment]
It seems like I was able to fix it by reverting to 1.8.0. Strange.

**Comment by weissk93** [comment]
I did not touch any of the settings. It just was not working. I seemed to get everything working somehow. Not sure what happened.

**Comment by chase9** [comment]
Did you get it working on the latest version or on the downgraded version you mentioned?ChasePlease excuse any grammatical errors as this was sent from my phone.On Jul 11, 2024, at 8:19 AM, weissk93 ***@***.***> wrote:﻿
I did not touch any of the settings. It just was not working. I seemed to get everything working somehow. Not sure what happened.

**Comment by weissk93** [comment]
No worries on grammar. Excuse my lack of technical understanding.

I think my Hoobs box was on its last legs. When I unplugged it and plugged it back in I could not get it to start back up. Rather than reflashing the SD Card or replacing it, I just decided to rebuild my entire setup on HomeBridge on my Mac mini.

Things were working at first on my Mac mini. Then I noticed that it would stop working randomly just like the Hoobs box. 

I tried downgrading the plugin and removing DNS settings on my Eero router. It seems the culprit was making sure my Mac mini did not fall asleep and "stayed on". First time using Homebridge on my Mac mini so rookie mistake.

However, not sure why it stopped working on the Hoobs box to begin with. I do not want to conflate the issues. In any case, things seem to be working fine. Homebridge runs alot better on more capable hardware.
